addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -232,14 +225,14 @@ func (r *AntreaConfigReconciler) getProviderServiceAccountName(clusterName strin
 	return fmt.Sprintf("%s-antrea", clusterName)
```

Comment:
There is also a method `func (r *AntreaConfigReconciler) getProviderServiceAccountName(clusterName string) ` in this file. I think we can also remove `(r *AntreaConfigReconciler)` from that method to make it a pure function, and then we move both `getProviderServiceAccountName` and `getNSXServiceAccountName` functions to antreaconfig_util.go.

Reply:
done.